### PR TITLE
Add 64_ansi and 64_iso Community Layouts

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

Adds ANSI and ISO versions of the 64-key 60% layout, with 2u Left Shift and five 1u keys and the end of the bottom row.

The layouts are similar to the [`60_ansi_arrow` layout](https://ae01.alicdn.com/kf/HTB1gnnoRFXXXXX8XFXXq6xXFXXXA.jpg?size=355038&height=517&width=750&hash=183d336a5d4bfd0354e2808a873de7a7) (KPrepublic XD60/XD64 pictured), but shrink the Left Shift from 2.25u to 2u so as to add the Forward Slash key back in to the Shift row.

Also adds a README file to the `layouts/default/` directory, containing box drawings of the Community Layouts that are supported.

## Types of Changes

- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: layouts/default/readme.md [hunk 1]
```diff
@@ -0,0 +1,704 @@
...
+│   │   │   │   │   │   │   │   │   │   │   │   │   │   │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+│      │   │   │   │   │   │   │   │   │   │   │   │        │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┤
+│        │   │   │   │   │   │   │   │   │   │   │      │   │
+├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─┴───┤
+│▒▒▒▒▒│   │     │                           │     │   │▒▒▒▒▒│
+└─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘
```
Comment (fauxpark) [inline layouts/default/readme.md:94]:
```suggestion
└─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─┴───┘
      │   │     │                           │     │   │
      └───┴─────┴───────────────────────────┴─────┴───┘
```
?

File: layouts/default/readme.md [hunk 2]
```diff
@@ -0,0 +1,704 @@
...
+│   │   │   │   │   │   │   │   │   │   │   │   │   │       │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┼───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │   │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+│      │   │   │   │   │   │   │   │   │   │   │   │        │   │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+│        │   │   │   │   │   │   │   │   │   │   │      │   │   │
+├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬─┬───┼───┼───┤
+│    │    │    │                        │    │    │▒│   │   │   │
+└────┴────┴────┴────────────────────────┴────┴────┴─┴───┴───┴───┘
```
Comment (fauxpark) [inline layouts/default/readme.md:216]:
```suggestion
│    │    │    │                        │    │    │ │   │   │   │
└────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
```

File: layouts/default/readme.md [hunk 3]
```diff
@@ -0,0 +1,704 @@
...
+│   │   │   │   │   │   │   │   │   │   │   │   │   │   │   │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┼───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │   │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+│      │   │   │   │   │   │   │   │   │   │   │   │        │   │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+│        │   │   │   │   │   │   │   │   │   │   │      │   │   │
+├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬─┬───┼───┼───┤
+│    │    │    │                        │    │    │▒│   │   │   │
+└────┴────┴────┴────────────────────────┴────┴────┴─┴───┴───┴───┘
```
Comment (fauxpark) [inline layouts/default/readme.md:231]:
```suggestion
│    │    │    │                        │    │    │ │   │   │   │
└────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
```

File: layouts/default/readme.md [hunk 4]
```diff
@@ -0,0 +1,704 @@
...
+│   │   │   │   │   │   │   │   │   │   │   │   │   │       │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┼───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │   │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+│      │   │   │   │   │   │   │   │   │   │   │   │        │   │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+│        │   │   │   │   │   │   │   │   │   │   │      │   │   │
+├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬─┬───┼───┼───┤
+│     │   │     │                           │     │▒│   │   │   │
+└─────┴───┴─────┴───────────────────────────┴─────┴─┴───┴───┴───┘
```
Comment (fauxpark) [inline layouts/default/readme.md:246]:
```suggestion
│     │   │     │                           │     │ │   │   │   │
└─────┴───┴─────┴───────────────────────────┴─────┘ └───┴───┴───┘
```

File: layouts/default/readme.md [hunk 5]
```diff
@@ -0,0 +1,704 @@
...
+│   │   │   │   │   │   │   │   │   │   │   │   │   │       │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┼───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │   │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    ├───┤
+│      │   │   │   │   │   │   │   │   │   │   │   │   │    │   │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴┬───┼───┤
+│        │   │   │   │   │   │   │   │   │   │   │      │   │   │
+├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬─┬───┼───┼───┤
+│    │    │    │                        │    │    │▒│   │   │   │
+└────┴────┴────┴────────────────────────┴────┴────┴─┴───┴───┴───┘
```
Comment (fauxpark) [inline layouts/default/readme.md:276]:
```suggestion
│    │    │    │                        │    │    │ │   │   │   │
└────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
```

File: layouts/default/readme.md [hunk 6]
```diff
@@ -0,0 +1,704 @@
...
+│   │   │   │   │   │   │   │   │   │   │   │   │   │       ││   │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤├───┼───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     ││   │   │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤└───┴───┘
+│      │   │   │   │   │   │   │   │   │   │   │   │        │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤┌───┐
+│        │   │   │   │   │   │   │   │   │   │   │          ││   │
+├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬─┬──┴┼───┼───┐
+│    │    │    │                        │    │    │    │▒│   │   │   │
+└────┴────┴────┴────────────────────────┴────┴────┴────┘ └───┴───┴───┘
```
Comment (fauxpark) [inline layouts/default/readme.md:321]:
```suggestion
│    │    │    │                        │    │    │    │ │   │   │   │
└────┴────┴────┴────────────────────────┴────┴────┴────┘ └───┴───┴───┘
```

File: layouts/default/readme.md [hunk 7]
```diff
@@ -0,0 +1,704 @@
...
+│   │   │   │   │   │   │   │   │   │   │   │   │   │       ││   │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤├───┼───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     ││   │   │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │└───┴───┘
+│      │   │   │   │   │   │   │   │   │   │   │   │   │    │
+├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤┌───┐
+│    │   │   │   │   │   │   │   │   │   │   │   │          ││   │
+├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬─┬──┴┼───┼───┐
+│    │    │    │                        │    │    │    │▒│   │   │   │
+└────┴────┴────┴────────────────────────┴────┴────┴────┘ └───┴───┴───┘
```
Comment (fauxpark) [inline layouts/default/readme.md:336]:
```suggestion
│    │    │    │                        │    │    │    │ │   │   │   │
└────┴────┴────┴────────────────────────┴────┴────┴────┘ └───┴───┴───┘
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: layouts/default/readme.md (hunk 1 above)
Comment (noroadsleft) [reply layouts/default/readme.md:94]: I actually had it this way originally, but then I thought it looked "off," somehow.

Comment (fauxpark) [reply layouts/default/readme.md]: I like having the non-rectangular shapes, it shows off what you can do with box drawing characters 😄

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Keyboards I can find that support this layout:

- DZ60
- ACR60
- XD60 (rev2 and rev3)
- [Melgeek MJ64](https://github.com/qmk/qmk_firmware/pull/10425)
- Longnald Corin
- ORG60
- Panc60
- Play Keyboard PK60
- YD60MQ
- YMDK Bface

and possibly:

- ALF DC60
- BIOI G60BLE
- DO60
- Exclusive E6-V2 LE
- Mechkeys MK60

**Review by mrsendyyk — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Comment by zvecr** [comment]
:tada:

**Pull request merged**